Makefile

```diff
@@ -64,7 +64,7 @@ gtest-compile: CXXFLAGS = --verbose -Wall -O0 -g -c $(gtest_home)/src/gtest-all.
           -I$(gtest_home)/include
 gtest-compile: 
 	${info Building gtest library}
-	$(CXX) ${CXXFLAGS} $@.cc -o $@
```

Comment:
`-o` is definite in https://github.com/danbev/learning-v8/pull/21/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L61